server/models.js

```diff
@@ -0,0 +1,22 @@
+// const connectionString = process.env.PG_URI;
+// const pgp = require('pg-promise')();
+// const db = pgp(connectionString);
+
+// module.exports = {db};
+
+const {Pool} = require('pg');
+require('dotenv').config();
```

Comment:
server.js takes care of configuring process.env so you should be able to remove this.